README.md

```diff
@@ -86,8 +86,8 @@ The following sections provide [the version integer](https://beacon-biosignals.g
     - `stop` (`Duration` w/ `NANOSECOND` unit): The stop offset in nanoseconds (exclusive) from the beginning of the recording. This value must be greater than `start`.
 - `kind` (`Utf8`): A string identifying the kind of signal that the row represents. Valid `kind` values are alphanumeric, nonempty, lowercase, `snake_case`, and contain no whitespace, punctuation, or leading/trailing underscores.
 - `channels` (`List` of `Utf8`): A list of strings where the `i`th element is the name of the signal's `i`th channel. A valid channel name...
-    - ...conforms to the same format as `kind` (alphanumeric, nonempty, lowercase, `snake_case`, and contain no whitespace, punctuation, or leading/trailing underscores).
-    - ...conforms to an `a-b` format where `a` and `b` are valid channel names. Furthermore, to allow arbitrary cross-signal referencing, `a` and/or `b` may be channel names from other signals contained in the recording. If this is the case, such a name must be qualified in the format `signal_name.channel_name`. For example, an `eog` signal might have a channel named `left-eeg.m1` (the left eye electrode referenced to the mastoid electrode from a 10-20 EEG signal).
+    - ...is alphanumeric, nonempty, lowercase, `snake_case`, and contain no whitespace, punctuation, or leading/trailing underscores. Additional allowed characters include: `-`, `+`, `(`, `)`, `/`, `.`. If parentheticals are used, a matching closing parenthetical must be included for every open parenthetical included. 
```

Comment:
yes and yeah, probably

Reply:
Just in case you aren't joking...
```julia
function parens_balanced(s)
    n_open = n_closed = 0
    for c in s
        if c === '('
            n_open += 1
        elseif c === ')'
            n_open > 0 || return false
            n_closed += 1
        end
    end
    return n_open == n_closed
end
```

Reply:
That gives
```julia
julia> parens_balanced("())(")
true
```
I think we want
```julia
julia> function parens_balanced2(s)
           depth = 0
           for c in s
               if c === '('
                   depth += 1
               elseif c === ')'
                   depth -= 1
               end
               depth < 0 && return false
           end
           return depth == 0
       end
parens_balanced2 (generic function with 1 method)

julia> parens_balanced2("())(")
false

julia> parens_balanced2("()()")
true

julia> parens_balanced2("(())()")
true

julia> parens_balanced2("((()()))")
true
```

Reply:
@ararslan will find any excuse to play a good ol' round of code golf :)

Reply:
Weeeeeell code golf would be more like
```julia
s->(d=0;for c=s d+=(c=='(' ? 1 : c==')' ? -1 : 0);d<0&&return 1==0end;d==0)
```
for 75 bytes. Though you might be able to do something with `(` and `)` being ASCII codes 40 and 41, respectively, along with `(-1)^(c&1)` being the appropriate increment or decrement if you know that `c` is 40 or 41. The loop start would need to be `c::Int=s` instead of `c=s` so it may end up a wash anyway.